github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml

```diff
@@ -94,37 +65,6 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-workflow-k8s.sh"
-    - name: pull-e2e-cluster-network-addons-operator-workflow-okd
```

Comment:
Please remove these changes from this PR. I'd be happy to take them in right away on a different PR.